.buildkite/pipeline-utils/ci-stats/pick_test_group_run_order.ts

```diff
@@ -561,3 +561,50 @@ export async function pickTestGroupRunOrder() {
     ].flat()
   );
 }
+
+export async function pickScoutTestGroupRunOrder(scoutConfigsPath: string) {
+  const bk = new BuildkiteClient();
+  const envFromlabels: Record<string, string> = collectEnvFromLabels();
+
+  const rawScoutConfigs = JSON.parse(Fs.readFileSync(scoutConfigsPath, 'utf-8'));
+  const pluginsWithScoutConfigs: string[] = Object.keys(rawScoutConfigs);
+
+  const scoutGroups = pluginsWithScoutConfigs.map((plugin) => ({
+    title: plugin,
+    key: plugin,
+    usesParallelWorkers: rawScoutConfigs[plugin].usesParallelWorkers,
+    group: rawScoutConfigs[plugin].group,
+  }));
+
+  // upload the step definitions to Buildkite
+  bk.uploadSteps(
```

Comment:
Does it provide us with any value to upload an empty array of steps?

I ask this because I was thinking we could simplify the code here by removing the ternary operator and only calling `bk.uploadSteps` if we have any steps to upload.